pages/docs/deploy/render.mdx

```diff
@@ -0,0 +1,61 @@
+# Render
+
+[Render](https://render.com) lets you easily deploy and scale full stack applications. You can deploy your Inngest functions on Render using any web framework, including [Next.js](https://docs.render.com/deploy-nextjs-app), [Express](https://docs.render.com/deploy-node-express-app), and [FastAPI](https://docs.render.com/deploy-fastapi).
+
+
+
+Below, we'll cover how to deploy:
+1. A production Inngest app
+1. Preview apps for each of your Git development branches
+
+### Before you begin
+* Create a web application that serves Inngest functions.
+* Test this web app locally with the [Inngest dev server](/docs/dev-server).
+
+## Deploy a production app on Render
+1. Deploy the web application that contains your Inngest functions to Render.
+    * See [Render's guides](https://docs.render.com) to learn how to deploy specific frameworks, such as:
+        - [Next.js](https://docs.render.com/deploy-nextjs-app)
+        - [Express](https://docs.render.com/deploy-node-express-app)
+        - [FastAPI](https://docs.render.com/deploy-fastapi)
+1. Set the `INNGEST_SIGNING_KEY` and `INNGEST_EVENT_KEY` environment variables on your Render web app.
+    * You can easily [configure environment variables](https://docs.render.com/configure-environment-variables) on a Render service through the Render dashboard.
+    * You can find your production `INNGEST_SIGNING_KEY` [here](https://app.inngest.com/env/production/manage/signing-key), and your production `INNGEST_EVENT_KEY`s [here](https://app.inngest.com/env/production/manage/keys).
+1. Manually sync your Render web app with Inngest.
+    * See [this Inngest guide](/docs/apps/cloud) for instructions.
+
+## Automatically sync your app with Inngest
+Each time you push changes to your Inngest functions, you need to sync your web app with Inngest.
+
+For convenience, you can automate these syncs. Here are two possible approaches:
+
+1. **Self-sync**: Modify your app's code so that your app calls its own Inngest `serve` endpoint after it starts up. (In other words, make your app issue the ["curl command"](/docs/apps/cloud#curl-command) on itself inside of your app's entry point.)
+2. **CI/CD**: Another option is to make your CI/CD pipeline sync your app using the ["curl command"](/docs/apps/cloud#curl-command) after each deploy to Render.
```

Comment:
@jess-render, by any chance, do you have some Github Actions snippet featuring such `curl` action?

If not, I can provide one!

Reply:
Happy to have you contribute!

Reply:
@charlypoly Another thought -- this topic (CI/CD) seems like it could be covered in a general section, e.g. in https://www.inngest.com/docs/apps/cloud 

I'd be happy to contribute a separate section, then link to it here afterward.

Reply:
(Also this is Jess, just on my personal device right now. I'm at the hospital to support a family member.)

Reply:
Hi again @jssln!
Sorry for the delay here.

Let's merge as it is, and I'll add some screenshots along with a GH Action YAML example using your https://github.com/marketplace/actions/render-deploy-action

Reply:
Thanks Charly!

A heads up, we're starting work on webhooks, including post-deploy hooks: https://feedback.render.com/features/p/webhooks-1

So we will be able to update this section, and potentially work on a better integration (similar to Vercel's).